### PR TITLE
Fix react-router dependency. Fix #439.

### LINKED ISSUE #439: Cannot resolve module 'qs' and 'deep-equal' in react-router
Fresh clone. For the todo example, run `npm i` and `npm start`. Here is the error message:

```
  [761] ./js/queries/ViewerQueries.js 995 bytes {0} [built]

ERROR in ./~/react-router/~/history/lib/useQueries.js
Module not found: Error: Cannot resolve module 'qs' in /Users/octref/Code/learn/relay/relay/examples/todo/node_modules/react-router/node_modules/history/lib
 @ ./~/react-router/~/history/lib/useQueries.js 11:10-23

ERROR in ./~/react-router/~/history/lib/createHistory.js
Module not found: Error: Cannot resolve module 'deep-equal' in /Users/octref/Code/learn/relay/relay/examples/todo/node_modules/react-router/node_modules/history/lib
 @ ./~/react-router/~/history/lib/createHistory.js 13:17-38
webpack: bundle is now VALID.
```

Manually installing `qs` and `deep-equal` solve the problem. But I guess that's not the way the problem should be resolved.

I'm using Node v4.1.2 and npm 2.14.4 if that matters.

### PR DESCRIPTION
**Pull request by octref**

### CODE REVIEW

File: examples/todo/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "react": "^0.14.0-rc",
     "react-dom": "^0.14.0-rc",
     "react-relay": "file:../../",
-    "react-router": "^1.0.0-rc1",
```
Comment (zpao) [inline examples/todo/package.json]: I think this *should* install rc3 as is (at least with a fresh install or `npm update`)

Comment (octref) [inline examples/todo/package.json]: @zpao Do you mean make it precisely `1.0.0-rc3`?

Comment (zpao) [inline examples/todo/package.json]: No, I think this is fine, I just misunderstood the reason for the change.

**Comment by kassens** [comment]
@facebook-github-bot import

**Comment by octref** [comment]
@kassens 
There is a merge conflict because of https://github.com/facebook/relay/commit/89d62360d5133e43ec16500f1407d48792c6704b

**Comment by kassens** [comment]
@octref: Yes, saw that, but was easy to resolve.

**Pull request closed without merging**